hip_data_tools/google/adwords.py

```diff
@@ -579,3 +583,69 @@ def set_query_to_fetch_all(self, start_index: int = 0, page_size: int = 500):
                  .Limit(start_index, page_size)
                  .Build())
         self.set_query(query)
+
+
+class AdWordsReportDefinitionReader(AdWordsUtil):
+    """
+    Class to interact with the ReportDefinitionService and access fields for adwords reports
+    Args:
+        conn (GoogleAdWordsConnectionManager): Connection manager to handle the creation of
+    adwords client
+    """
+
+    def __init__(self, conn: GoogleAdWordsConnectionManager):
+        super().__init__(conn, "ReportDefinitionService", "v201809")
+
+    def get_report_fields(self, report_type: str) -> List[dict]:
+        """
+        Get a list of fields for a given report.
+        Args:
+            report_type (str): Possible reports are defined in the documentation here -
+            https://developers.google.com/adwords/api/docs/appendix/reports/all-reports
+        Returns: List[dict]
+        """
+        return self._get_service().getReportFields(report_type)
```

Comment:
lets add the parameter name, like
getReportFields(report_type=report_type)

Reply:
it might be some kind of a service call without a keyword arg, and only a positional arg. https://github.com/googleads/googleads-python-lib/search?q=getReportFields&unscoped_q=getReportFields